src/lib/features/playground/playground-service.ts

```diff
@@ -42,34 +50,49 @@ export class PlaygroundService {
         projects: typeof ALL | string[],
         environments: string[],
         context: SdkContextSchema,
-    ): Promise<PlaygroundFeatureSchema[]> {
+    ): Promise<any> {
         const segments = await this.segmentService.getActive();
         const environmentFeatures = await Promise.all(
             environments.map((env) => this.resolveFeatures(projects, env)),
         );
         const contexts = generateObjectCombinations(context);
 
         const results = await Promise.all(
-            environmentFeatures.flatMap(({ features, featureProject }) =>
-                contexts.map((singleContext) =>
-                    this.evaluate({
-                        features,
-                        featureProject,
-                        context: singleContext,
-                        segments,
-                    }),
-                ),
+            environmentFeatures.flatMap(
+                ({ features, featureProject, environment }) =>
+                    contexts.map((singleContext) =>
+                        this.evaluate({
+                            features,
+                            featureProject,
+                            context: singleContext,
+                            segments,
+                            environment,
+                        }),
+                    ),
             ),
         );
-        return results.flat();
+        const items = results.flat();
+        const itemsByName = groupBy(items, (item) => item.name);
+        return Object.entries(itemsByName).map(([name, entries]) => {
+            const groupedEnvironments = groupBy(
+                entries,
+                (entry) => entry.environment,
+            );
+            return {
+                name,
+                projectId: entries[0]?.projectId,
```

Comment:
we can take any entry in the group so I take the first one since it's always there